File: exercicio03/python/main.py
from fazenda2 import Fazenda2
from figura2 import Figura2


f_01 = Figura2(2, 5)
f_01.desenhar()
f_01.pintar()

print("_"*55)

fazenda = Fazenda2("Haras Manga_Larga", 2000, "Fabio", "batata")
fazenda.imprimi()
fazenda.adubar()
fazenda.plantar()
print("\n")
fazenda.imprimi()

File: exercicio03/java/Main.java
package exercicio03.java;

public class Main {
  public static void main(String[] args) {
    Figura2 f_01 = new Figura2(2, 5); // instancia_Figura

    f_01.desenhar();
    f_01.pintar();
    f_01.imprimi();

    System.out.println("______________________________________________________\n");

    Fazenda2 fzd = new Fazenda2("Haras Manga_Larga", 2000, "Fábio", "batata"); // instancia_Fazenda

    fzd.imprimi();
    fzd.adubar();
    fzd.plantar();
  }
}
